src/scripts/form-validation.js

```diff
@@ -0,0 +1,105 @@
+const emailEl = document.querySelector('#email');
+const userName = document.querySelector('#Name');
+const userLastName = document.querySelector('#Last_Name');
+const message = document.querySelector('#message');
+const form = document.querySelector('#Form-form');
+
+const isEmailValid = (email) => {
+  const re = /^[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$/;
+  return re.test(email);
+};
```

Comment:
As requested by the previous code reviewer, when I tried to submit the form with the email TestR@mail.com, it actually did accept the value, and the form was submitted, which should not be the case as per validation requirements. Kindly fix the issue.

Reply:
**I don't understand how this is possible**

Reply:
# So what is the suggestion of how I suppose to fix it?